cmake/FindCppcoroCoroutines.cmake

```diff
@@ -0,0 +1,36 @@
+include(CheckCXXCompilerFlag)
+include(CheckIncludeFileCXX)
+include(FindPackageHandleStandardArgs)
+
+check_cxx_compiler_flag(/await Coroutines_SUPPORTS_MS_FLAG)
+check_cxx_compiler_flag(-fcoroutines-ts Coroutines_SUPPORTS_GNU_FLAG)
```

Comment:
I think that since the adoption of coroutines into C++20 that clang (trunk?) now also enables coroutines when `-std=c++2a` is enabled.